rules/rules.libsonnet

```diff
@@ -21,128 +21,128 @@
-          },
-          {
-            record: 'job:ceph_mds_metadata:count',
-            expr: |||
-              count(ceph_mds_metadata{%(cephExporterSelector)s})
-            ||| % $._config,
-          },
-          {
-            record: 'job:ceph_mon_metadata:count',
-            expr: |||
-              count(ceph_mon_metadata{%(cephExporterSelector)s})
-            ||| % $._config,
-          },
           {
             record: 'job:ceph_osd_metadata:count',
             expr: |||
               count(ceph_osd_metadata{%(cephExporterSelector)s})
             ||| % $._config,
           },
           {
-            record: 'job:ceph_mgr_metadata:count',
-            expr: |||
-              count(ceph_mgr_metadata{%(cephExporterSelector)s})
-            ||| % $._config,
-          },
-          {
-            record: 'job:ceph_rgw_metadata:count',
-            expr: |||
-              count(ceph_rgw_metadata{%(cephExporterSelector)s})
-            ||| % $._config,
-          },
-          {
-            record: 'job:ceph_mon_metadata:distinct',
-            expr: |||
-              count(count(ceph_mon_metadata{%(cephExporterSelector)s}) by (ceph_version))
-            ||| % $._config,
-          },
-          {
-            record: 'job:ceph_mgr_metadata:distinct',
+            record: 'job:kube_pv:count',
             expr: |||
-              count(count(ceph_mgr_metadata{%(cephExporterSelector)s}) by (ceph_version))
+              count(kube_persistentvolume_info)
             ||| % $._config,
           },
           {
-            record: 'job:ceph_osd_metadata:distinct',
+            record: 'job:ceph_pools_iops:total',
             expr: |||
-              count(count(ceph_osd_metadata{%(cephExporterSelector)s}) by (ceph_version))
+              sum(ceph_pool_rd{%(cephExporterSelector)s}+ ceph_pool_wr{%(cephExporterSelector)s})
             ||| % $._config,
           },
           {
-            record: 'job:ceph_mds_metadata:distinct',
+            record: 'job:ceph_pools_iops_bytes:total',
             expr: |||
-              count(count(ceph_mds_metadata{%(cephExporterSelector)s}) by (ceph_version))
+              sum(ceph_pool_rd_bytes{%(cephExporterSelector)s}+ ceph_pool_wr_bytes{%(cephExporterSelector)s})
             ||| % $._config,
           },
           {
-            record: 'job:ceph_rgw_metadata:distinct',
+            record: 'job:ceph_versions_running:count',
             expr: |||
-              count(count(ceph_rgw_metadata{%(cephExporterSelector)s}) by (ceph_version))
+              count(count(ceph_mon_metadata{%(cephExporterSelector)s} or ceph_osd_metadata{%(cephExporterSelector)s} or ceph_rgw_metadata{%(cephExporterSelector)s} or ceph_mds_metadata{%(cephExporterSelector)s} or ceph_mgr_metadata{%(cephExporterSelector)s}) by(ceph_version))
             ||| % $._config,
           },
         ],
       },
     ],
   },
 }
+          // {
+          //   record: 'job:ceph_osd_in:count',
```

Comment:
You want to keep these lines commented?
Remove this.

Reply:
I would say, lets have these commented and be here. We might need some of these later

Reply:
Ok. Still looks bad. Anyway.